include/ibm/management_console_rest.hpp

```diff
@@ -953,6 +955,7 @@ inline void getCSREntryAck(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
             {
                 asyncResp->res.result(boost::beast::http::status::bad_request);
                 asyncResp->res.jsonValue["Description"] = "Bad CSR";
+                BMCWEB_LOG_ERROR << "SignCSR failed with Bad CSR";
```

Comment:
This one is a ERROR but ones below are a INFO ? Why is this one a ERROR ?

Reply:
this hits when user provides  invalid CSR